profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyNamePronunciationEdit.html

```diff
@@ -170,6 +170,7 @@
             audio.muted = true;
             audio.srcObject = microphone;
             var options = {
+                sampleRate: 43100,
```

Comment:
this seems odd

Reply:
I remember this was reported to me by NYU several years ago, they mentioned the pitch issue and I remember changing this parameter, maybe I only changed the param in their branch.

This sample rate doesn't look very standard to me, maybe we should fit to an standard one?

Reply:
yeah i think 44100 is the standard?

Reply:
Yes, it seems that 44100 is the standard, but in my testing it seems a little high yet.

Changed to 44100.

Tested in Chrome, Edge, Firefox and Opera.

Reply:
After setting to 44100, how does your voice sound?

Reply:
@mpellicer 
After setting to 44100 my voice still sounds natural. I think that there's little difference between the trunk and 44100, just a bit, but enough. In short, now there is no high-pitched feel. And I think that's the important question.